bibliopixel/animation/collection.py

```diff
@@ -1,11 +1,13 @@
 from .. project import aliases, project
 from . import animation, runner
+import traceback
+from .. import log
 
 
 class Collection(animation.BaseAnimation):
-    def __init__(self, layout, animations=None):
+    def __init__(self, layout, animations=None, no_fail=False):
```

Comment:
I realized that Collection broke my graceful fail logic that would allow remote to run even if an animation didn't load correctly. So I needed a way to tell _make_animation to not freak out on a failed load.